README.md

```diff
@@ -52,6 +52,19 @@ See the documentation for the
 [Rails integration class](http://www.rubydoc.info/gems/opencensus/OpenCensus/Trace/Integrations/Rails)
 for more information.
 
+### Getting started with Sidekiq
```

Comment:
IMO your implementation is achieving something bigger than Sidekiq. You may want to either split this up or update the readme so that it notifies the user you are doing integration for other Rails components such as Active Record.

Reply:
I updated the documentation in this part of the README following the separation of the ActiveSupport integration from the Sidekiq integration. If you have any comments on it, feel free to post them here.